aiodataloader.py

```diff
@@ -117,7 +170,7 @@ def load_many(self, keys):
 
         return gather(*[self.load(key) for key in keys])
 
-    def clear(self, key):
+    def clear(self, key: KeyT) -> "DataLoader":
```

Comment:
I'm pretty sure a TypeVar is needed here, like:
```
def clear(self: DataLoaderT, key: KeyT) -> DataLoaderT:
```

That TypeVar would be something like:
```
DataLoaderT = TypeVar("DataLoaderT", bound="DataLoader[Any, Any]")
```

Reply:
Good insight, I'll put that in

Reply:
Actually, I don't know what pyright etc will infer when returning self, lemme do an experiment

Reply:
Can confirm, it's DataLoader[Unknown, Unknown], which honestly seems like something pyright/etc should be able to figure out on its own but I'm finding that life in Python types is like that